backend/canvas_app_explorer/serializers.py

```diff
@@ -12,13 +12,24 @@ class Meta:
         fields = '__all__'
 
 class LtiToolSerializer(serializers.ModelSerializer):
-    # Addiitonal expanded ead only field to make it easier on the front-end
-    canvas_placement_expanded = CanvasPlacementSerializer(read_only=True, many=True, source="canvas_placement")
-    navigation_enabled = serializers.SerializerMethodField()
+    """
+    Serializer for LtiTool model, with nested CanvasPlacements
+    """
+    canvas_placement_expanded = CanvasPlacementSerializer(read_only=True, many=True, source='canvas_placement')
 
     class Meta:
         model = models.LtiTool
-        fields = '__all__'
+        fields = [
+            'name', 'canvas_id', 'logo_image', 'logo_image_alt_text', 'main_image',
+            'main_image_alt_text', 'short_description', 'long_description', 'privacy_agreement',
+            'support_resources', 'canvas_placement_expanded'
+        ]
+
+class LtiToolWithNavSerializer(LtiToolSerializer):
+    """
+    Serializer extending LtiToolSerializer with additional navigation data specific to a course context
+    """
```

Comment:
I refactored this with the idea that we may need a separate model-only serializer for #27. There was also a `canvas_placement` field that wasn't being used (since we're traversing the M:M to create nested data instead), so I specified the fields we wanted (almost all of them).